meetings/2023-07-19.md

```diff
@@ -78,28 +88,73 @@ URL: https://github.com/solid/specification/pull/543
+* SC: Some of the aspects of outreach mentioned in the charter discussion can fall under this.
+* TBL: Solid Creators, are they part of the CG? 
+* SC: Is this a separate topic? 
+* TBL: You mentioned outreach. 
+* SC: Solid Creators is a a role in the Solid Team. Creators don't have a specific role in the CG, or hasn't been in the past. Creators in majority are not editors/authors of any specs or producing contributing guidelines, meeting material, or any of that. Some people with multiple hats, predominantly they are not listed in a spec as a solid creator.
+* TBL: Right it is mainly about the website. 
+* SC: That's one of many things they do yes. With respect to work items in the CG they don't have any special overview than anybody else. CG generally has editors/authors. 
+* SC: Whether forming a Business group is something folks want to pursue, we can discuss (possibly a dedicated meeting).
+* TBL: Do you mean a business group focused on particular verticals, like healthcare?
+* SC: I interpret it in general, yes, it could be covered. W3C says "application of technology to business contexts", so means gives input to CG/WG, and yes verticals of any kind, that would all be covered under business group. This is something W3C can help us guide. 
+* TBL: If you have a BG to look at all of those it does not ??? might be useful to have one for each domain, like healthcare, to focus around a topic.
+* SC: I just want to put it out there as something to look into, whether it's one or specific to domains. I agree with TBL.
+* eP: Is there a precedent that some effort organized as CG also has a BG? 
+* TBL: I think there is, that's why someone suggested it, because it's common.
+* eP: Would be good to know and see how their experience is, and how they divide the work. 
+* SC: If we can have a link, that'd be great. Raising this because it's an angle to that discussion. 
+
+
+### Solid Demos
+* SC: Background: Originally proposed back on 2021-08-05 at [Gitter/Matrix solid/app-development](https://matrix.to/#/!ogHmAuzhszmxlGQrVp:gitter.im/$FdBbaChBNThzgicpyNhs5eCPp07-Q1KCBrq8AqsTrio) . There is interest to conduct these series of events. Also welcomed by Solid Editors Team: https://github.com/solid/specification/blob/main/meetings/2021-09-22.md#solid-demos .
 * SC: Quoting for convenience/discussion:
   >@/all Who'd be interested in a regular "Solid Demos" gathering for the community to showcase their implementations (servers, apps..)? :)
+  >
   >Think along the lines of a demoscene ( https://en.wikipedia.org/wiki/Demoscene ) for Solid. It could be pretty useful to increase the visibility of implementations as well as to attract feedback from (developers, users, spec developers, ... investors?).
```

Comment:
```suggestion
* SC: Background: Originally proposed back on 2021-08-05 at [Gitter/Matrix solid/app-development](https://matrix.to/#/!ogHmAuzhszmxlGQrVp:gitter.im/$FdBbaChBNThzgicpyNhs5eCPp07-Q1KCBrq8AqsTrio). There is interest to conduct these series of events. Also welcomed by Solid Editors Team: <https://github.com/solid/specification/blob/main/meetings/2021-09-22.md#solid-demos>.
* SC: Quoting for convenience/discussion:
  >@/all Who'd be interested in a regular "Solid Demos" gathering for the community to showcase their implementations (servers, apps..)? :)
  >
  >Think along the lines of a [demoscene](https://en.wikipedia.org/wiki/Demoscene) for Solid. It could be pretty useful to increase the visibility of implementations as well as to attract feedback from (developers, users, spec developers, ... investors?).
```